docs/virtual_cap_pools.md

```diff
@@ -0,0 +1,137 @@
+# Virtual Cap Pools
+
+## Overview
+
+Before virtual cap pools were implemented, each school would have a `SchoolDeviceAllocation` for devices and possibly another `SchoolDeviceAllocation` for routers.  The `SchoolDeviceAllocation` contains the `allocation`, `cap` (the amount of their allocation they can currently order) and a `devices_ordered` value indicating how much of the `cap` had been used.
+
+The `cap` values are sent automatically to Computacenter so that they are able to control the amount available for the school to order via Techsource.  The `devices_ordered` values are updated by Computacenter once a school has placed an order.
+
+For a responsible body with many centrally managed schools, there became a need to be able to control where devices were delivered in order to facilitate centralised IT management for the organisation.
```

Comment:
I think we could maybe do with a little bit more on this point - what problems did it cause, for whom?

Reply:
Thanks, I've reworked this bit.